msticpy/nbtools/security_alert.py

```diff
@@ -9,6 +9,7 @@
 from typing import Dict, Any, List
 
 import pandas as pd
+from deprecated.sphinx import deprecated
```

Comment:
Don't you need to import a replacement for SecurityAlert here?
e.g. from ..datamodel.soc import SentinelAlert as SecurityAlert

Ah ok - so SentinelAlert isn't replacing SecurityAlert? Is that too traumatic a shift?

Reply:
Maybe we can do that in phase 2